lib/manageiq/providers/openstack/legacy/events/openstack_stf_event_monitor.rb

```diff
@@ -0,0 +1,112 @@
+require 'manageiq/providers/openstack/legacy/openstack_event_monitor'
+require 'manageiq/providers/openstack/legacy/events/openstack_event'
+require 'manageiq/providers/openstack/legacy/events/openstack_stf_event_converter'
+require 'manageiq/providers/openstack/legacy/events/openstack_stf_event_receiver'
+require 'manageiq/providers/openstack/legacy/events/openstack_stf_event_test_receiver'
+require 'qpid_proton'
```

Comment:
will this all be eager-loaded even if not needed?

Reply:
This will be loaded when an OpenStack Event monitor is needed (required by manager_mixin methods https://github.com/ManageIQ/manageiq-providers-openstack/blob/master/app/models/manageiq/providers/openstack/manager_mixin.rb)